pkg/kapp/cmd/app/delete.go

```diff
@@ -149,11 +149,26 @@ func (o *DeleteOptions) Run() error {
 		return err
 	}
 
+	if !shouldFullyDeleteApp {
+		defer func() {
+			_, numDeleted, _ := app.GCChanges(ctlapp.AppChangesMaxToKeepDefault, nil)
```

Comment:
We should probably revisit this default (200 right now I believe) at some point 🤔